proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+ <lynX> Patchworky proposal. Is there any scientific sociologic backing to doing things this way? Liquid democracy at least has http://arxiv.org/abs/1503.07723 speaking for it. Also, representation doesn't work.
+We consider the harmonization of corporate taxation in Europe as an effective and even indispensable measure to ensure justice of balance in Europe.
+ <lynX> Probably true, but do we have scientific material on the subject? And does it belong to this pillar?
+On point 5, we discovered that some solutions practiced by politics and authorities in Austria could actually be considered as an example: Funds, co-founding, investments, tax advantages/reduction for sustainable projects.
+It has been said that the practical support for such projects (logistics, office etc.) shall be supported by the public hand.
+We highly recommend the structures of the “economy for the common good” initiated by the Austrian Christian Felber. Projects will be evaluated by a multi-step process including the examination of common good impact /-orientation, feasibility etc. More information: www.ecogood.org/en/
+On question 10, a member underlined the urgent need for hard gender quotas for professors.
+ <lynX> Off-topic?
+
+# 3 
+# 30.06.2018 Proposal for reform of education media- and technology-related subjects and topics in School, university as well as the public sector like libraries - 18-06-30
+
+Regarding the changes in technology and society influenced by digitization, the internet, social media etc. the educational sector has to adopt existing knowledge and practices to teach children, youth and students skills and understanding to navigate in the contemporary digital world.
+As has been shown the technological means that have come into existence in the last decades have been exploited by the industrial-entertainment-military complex as well as by states to undermine existing liberties and the maturity (Mündigkeit) of both societies and individuals. This is has to be acknowledged as existentially undemocratic by politicians, educators and media outlets alike. All actors representing civil society, democratic functions or values have to take on their “Bildungsauftrag”, which has been dramatically neglected until now to foster democratic technologies. By this we mean software and hardware solutions that aren't corrupted by neocapitalist, geopolitical, military means. For instance this can mean the following:
```

Comment:
> industrial-entertainment-military complex 

We should avoid such language in our paper as it is based on non-proven claims and is not helpful in establishing a public discourse - subliminal guy